### PR TITLE
Lesson 5 tasks 1-3

### PR DESCRIPTION
**Pull request by DergachovaAnna**
Done HW for lesson 5, tasks 1-3

### CODE REVIEW

File: lesson_5.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# If placed as text then each line should contain only elements of one tuple
+# separated by spaces (eg "100 2 3"). The file must be created by a script in a pre-created \test\data directory tree.
+# The directory tree must be created by the script. Push only the code to the repository (not directories or file).
+import random
+import os
+import pickle
+from collections import Counter
+
+
+if __name__ == '__main__':
```
Comment (bohdandovbsyh) [inline lesson_5.py:20]: only one if __name__ == '__main__': 
can be in file

File: lesson_5.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    print(os.getcwd())
+    data_bytes = pickle.dumps(data)
+    with open('data.txt', 'w+b') as file:
+        result = file.write(data_bytes)
+
+
+if __name__ == '__main__':
+    with open('data.txt', 'rb') as file:
+        final_result = pickle.load(file)
+        for x, y, z in final_result:
```
Comment (bohdandovbsyh) [inline lesson_5.py:50]: what is x,y,z 
Please never use such naming

File: lesson_5.py (hunk 1 above)
Comment (DergachovaAnna) [reply lesson_5.py:20]: ok, thank you

File: lesson_5.py (hunk 2 above)
Comment (DergachovaAnna) [reply lesson_5.py:50]: improved, thank you

**Pull request merged**